Dockerfile

```diff
@@ -17,8 +17,6 @@ COPY controllers/ controllers/
 COPY config/ config/
 COPY pkg/ pkg/
 COPY service/ service/
-# Run tests and linting
-RUN make go-test
```

Comment:
it'd better to add info to the commit msg why this has to be removed and moved to sub target?